x-pack/legacy/plugins/apm/public/components/shared/Summary/getTransactionResultSummaryItem.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { idx } from '@kbn/elastic-idx';
+import { isRumAgentName } from '../../../../common/agent_name';
+import { Transaction } from '../../../../typings/es_schemas/ui/Transaction';
+import { HttpInfoSummaryItem } from './HttpInfoSummaryItem';
+import { ResultSummaryItem } from './ResultSummaryItem';
+
+const getTransactionResultSummaryItem = (transaction: Transaction) => {
```

Comment:
Since this is not re-used anywhere, perhaps better to keep it in `TransactionSummary`?

Reply:
Moved it to `TransactionSummary.tsx`.